ffw/NavigationRPC.js

```diff
@@ -416,6 +416,10 @@ FFW.Navigation = FFW.RPCObserver.create(
           }
           case 'Navigation.SetVideoConfig':
           {
+ 
+              var app_model = SDL.SDLController.getApplicationModel(request.params.appID)
```

Comment:
Some check for app_model validity should be great here))

Reply:
72bd13d